rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -512,7 +512,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 # This rule has stricter siblings:
 #   * 932331 (PL3)
 #
-SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx !-\d" \
```

Comment:
Add the comment about generating the regular expression (copy and adapt from another rule).